Header header
uint8 statusCode